src/app/contact/contactSectionMission.jsx

```diff
@@ -17,218 +17,218 @@ export default function ContactSectionMission({ sectionType }) {
   const [body, setBody] = useState("");
 
   const mailtoUri = `mailto:hello@openfresno.org?subject=${encodeURIComponent(
-    subject,
+    subject
   )}&body=${encodeURIComponent(body)}%0A%0A%0A%0A${encodeURIComponent(
-    firstName,
+    firstName
   )}%20${encodeURIComponent(lastName)}%0A${encodeURIComponent(email)}`;
 
   return (
     // <section className={`contact-section-${sectionType} contact-section-mission`}>
-    <section className={`app-color--${sectionType}-split py-12 lg:py-24`}>
-      <div
-        className={`page-container flex flex-col space-between lg:flex-row gap-16`}
-      >
-        <div className={`grow-1 basis-0 max-lg:mb-6`}>
-          <HeaderUnderline
-            title="Contact Form"
-            description="Get In Touch"
-            sectionType={sectionType}
-          >
+    <section className={`split-page max-lg:py-12`}>
+      <div className={`grow-1 basis-0 lg:mr-8 app-color--${sectionType} max-lg:px-8 pt-12 pb-18 lg:py-24`}>
```

Comment:
The mobile page margin should be 24px instead of 32px.
```suggestion
      <div className={`grow-1 basis-0 lg:mr-8 app-color--${sectionType} max-lg:px-6 pt-12 pb-18 lg:py-24`}>
```